### PR TITLE
feat(tutorial): complete step 4

### PR DESCRIPTION
**Pull request by lalasmuathasim**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/components/Info/_info.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.info-card__body {
+  margin-top: $spacing-06;
+  flex-grow: 1; // fill space so icons are bottom aligned
+  @include type-style('body-long-01');
+
+  // prevent large line lengths between small and medium viewports
+  @include carbon--breakpoint-between(321px, md) {
+    max-width: 75%;
+  }
+}
```
Comment (kellychurchill) [inline src/components/Info/_info.scss]: You're missing the `info-card__heading` class definition so your third word is not formatted correctly.

Comment (lalasmuathasim) [reply src/components/Info/_info.scss]: Happy to correct it. Can you be a bit more specific, what you mean by 'third word'

Comment (kellychurchill) [reply src/components/Info/_info.scss]: See this https://www.carbondesignsystem.com/tutorial/react-step-4#type

Comment (lalasmuathasim) [reply src/components/Info/_info.scss]: Hi i checked the code ... info-card__heading is defined already and the style is applied so the third word is already in bold.

please check [my deployed app](http://carbon-tutorial-lalasmuathasim.eu-gb.mybluemix.net/#/). I'm sorry if missed anything from the instruction

Comment (kellychurchill) [reply src/components/Info/_info.scss]: The class `info-card__heading` is not found in the file I am reviewing on GitHub. Have you pushed your updates to this PR?

Comment (lalasmuathasim) [reply src/components/Info/_info.scss]: Thanks for the quick response. I will do new pull request from https://github.com/lalasmuathasim/carbon-tutorial/pull/new/react-step-4 and go through the the steps 4 again and push to the step 4. I believe that is the step required to resolve this. Correct me if I'm wrong .. Thanks

Comment (lalasmuathasim) [reply src/components/Info/_info.scss]: I have updated the changes requested .. please review

**Pull request closed without merging**